packages/manager/modules/nutanix/src/dashboard/component/licence-tile/translations/Messages_de_DE.json

```diff
@@ -6,4 +6,4 @@
   "nutanix_cluster_licence_show_less": "Weniger Optionen anzeigen",
   "nutanix_cluster_licence_show_files_objects_size": "Inklusive (maximal {{size}} TB",
   "nutanix_cluster_licence_show_LCM": "Inklusive (nur Software Upgrade)"
-}
\ No newline at end of file
+}
```

Comment:
maybe roolback this space change so no translation file other that the french one is modified ?